example/src/App.tsx

```diff
@@ -13,6 +13,7 @@ import { Matrix } from "./Examples/Matrix";
 import { Aurora } from "./Examples/Aurora";
 import { Glassmorphism } from "./Examples/Glassmorphism";
 import { HomeScreen } from "./Home";
+import { Neumorphism } from "./Examples/Neumorphism/Neumorphism";
```

Comment:
Add new screens to the index-file in the examples folder to get a single import for this. I've fixed this in the Animation branch - it looks much cleaner.